controllers/controllers.js

```diff
@@ -50,3 +51,19 @@ exports.getCommentByArticleId = (req,res,next) => {
     });
 };
 
+//8
+exports.patchArticleById = (req,res,next) => {
+    const {article_id} = req.params;
+    const promises = [checkExists('articles', 'article_id',article_id), updateArticleVotes(article_id,req.body)];
```

Comment:
Really like the reusable checkExists function, we don't need to check this for a patch as PSQL will throw an error but this deffinitely works so don't worry about changing it now.